### PR TITLE
3main restart miner

### PR DESCRIPTION
**Pull request by papampi**
Changing the way 3main check for running miner and restart it
Some miners takes some time to exit, so after 3main send kill command, "running" check fails, and 3main wont start miner.

### CODE REVIEW

File: 3main [hunk 1]
```diff
@@ -1332,40 +1332,35 @@ EOF
...
+    then
+      echo "Miner output: ~/nvOC miner-log"
+      echo "    "
+      echo "    "
+    elif [ $LOCALorREMOTE == "LOCAL" ]
+    then
+      if [ $WTM_AUTO_SWITCH == "YES" ]
       then
-        echo "Miner output: 'nvOC miner-log'"
+        echo "Miner output: ~/nvOC miner-log"
```
Comment (LuKePicci) [inline 3main]: I removed ~ since nvOC was no more always into home folder

**Comment by LuKePicci** [comment]
@papampi have you already tested all `ps ax | grep ...` on running v2.1 tree? There was some issue also in minerinfo I had to fix, make sure there is not the same here.

**Comment by papampi** [comment]
@LuKePicci , Nope havent started going through all scripts yet, will do it when all edits finished.

File: 3main [hunk 2]
```diff
@@ -1332,40 +1332,35 @@ EOF
...
-    running=$(ps ax | grep -v cpuminer | grep -i screen | grep miner | awk '"miner" {print $1}')
-    if [ "$running" != "" ]
-    then
-      target=$(ps ax | grep -v cpuminer | grep -i screen | grep miner | awk '"miner" {print $1}')
-      kill "$target"
-    fi
-    sleep 2
+    while ps ax | grep SCREEN | grep -v cpuminer | grep -q miner ; do
+      echo "" | tee -a ${NVOC}/nvoc_logs/screenlog.0
+      echo "$(date) - Miner is running, 3main restarting miner" | tee -a ${NVOC}/nvoc_logs/screenlog.0
```
Comment (papampi) [inline 3main:1301]: @LuKePicci , You see any problems in these 2 echo?
I'm trying it on 19-2.0 , while loop kills miner, but does not echo to /home/m1/nvoc_logs/screenlog.0
Any idea?

Comment (papampi) [reply 3main:1301]: Got it, we have a "pkill -e miner" at the beginning of 3main, thats killing miner, not the while loop.

**Review by papampi — COMMENTED** [review]
I see no problems..
Merge?

**Review by LuKePicci — APPROVED** [review]

**Pull request merged**